install.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+set -e
+
+REPO="GoogleCloudPlatform/kubectl-ai"
+BINARY="kubectl-ai"
+
+# Detect OS
+OS="$(uname | tr '[:upper:]' '[:lower:]')"
+case "$OS" in
+  linux)   OS="linux" ;;
+  darwin)  OS="darwin" ;;
+  *) echo "Unsupported OS: $OS"; exit 1 ;;
```

Comment:
`kubectl-ai` supports windows as well

Reply:
Yea I don't have Windows machine to test (unless @mattn is interested 😄) I would change the wording a bit to clarify this method is for Linux & MacOS